ly/musicxml/lymus2musxml.py

```diff
@@ -92,22 +92,22 @@ def __init__(self):
 
     def parse_text(self, ly_text, filename=None):
         """Parse the LilyPond source specified as text.
-        
+
```

Comment:
Thank you. Actually you have to "break" it to make it consistent with our state of the code base ;-) I would really prefer all the code to be without these false-empty lines.